supported-devices/template.md

```diff
@@ -36,11 +36,11 @@ If your device isn't listed, it doesn't have support.
 :::
 
 ::: tip
-If you're on a smaller screen, scroll sideways to see whole table.
+If you are on a smaller screen, scroll sideways to see whole table.
 :::
 
 <AddScript script-url="../../supported-devices.js"/>
-<p>Search: <input type="text" class="deviceSearch"></p>
+<p><input type="text" class="deviceSearch" placeholder="Enter Your Boardname"></p>
```

Comment:
this implies that the search function only searches for board names, but that's not the case

Reply:
The search input in `docs/firmware/supported-devices.html` searches for boardnames only. This is not the search input that is found in the nav
<img width="671" alt="image" src="https://github.com/chrultrabook/docs/assets/72635727/a49f07c9-6d25-4dea-a742-98accd892616">
<img width="252" alt="image" src="https://github.com/chrultrabook/docs/assets/72635727/3a3b7df7-4029-44d7-8c8a-0adc9ad15351">

Reply:
No, it checks for boardname and device name

https://github.com/chrultrabook/docs/blob/020da2b85a571336c2473dd2fa2c2d35f4a175e0/supported-devices/search.js#L24

Reply:
I see, I will update the PR with the changes

Reply:
Added in latest commit